scripts/2009/01/sitemaps/sitemap.py

```diff
@@ -45,38 +45,50 @@ def xopen(filename):
     else:
         return open(filename)
 
+def urlsafe(name):
+    """Slugifies the name to produce OL url slugs
+
+    XXX This is duplicated from openlibrary.core.helpers because there
+    isn't a great way to import the methods from openlibrary as a
+    package
+    """
+    # unsafe chars according to RFC 2396
+    reserved = ";/?:@&=+$,"
+    delims = '<>#%"'
+    unwise = "{}|\\^[]`"
+    space = ' \n\r'
```

Comment:
Should handle tab as space too. Not sure about "_", which RFC2396 calls an unreserved mark, but it seems like its use would be asking for trouble.